modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -226,13 +229,14 @@ - (void) dontTestEnableConsoleLogging {
     // tag::console-logging[]
     CBLDatabase.log.console.domains = kCBLLogDomainAll; // <.>
     CBLDatabase.log.console.level = kCBLLogLevelVerbose; // <.>
-
     // end::console-logging[]
+}
 
-    // tag::console-logging-db[]
-    CBLDatabase.log.console.domains = kCBLLogDomainAll;
-
-    // end::console-logging-db[]
+- (void) dontTestEnableNewConsoleLogging {
+    // tag::new-console-logging[]
+    CBLConsoleLogSink* sink = [[CBLConsoleLogSink alloc] initWithLevel:kCBLLogLevelVerbose domains:kCBLLogDomainAll];
+    CBLLogSinks.console = sink;
```

Comment:
Could be in one line? (Same of the rest of the code snippets in this PR).

```
CBLLogSinks.console = [[CBLConsoleLogSink alloc] initWithLevel: kCBLLogLevelVerbose domains: kCBLLogDomainAll];
```

Also, it's a good idea  to consistently use the space or not using the pace after `:`.